lib/improver/nbhood/nbhood.py

```diff
@@ -149,14 +112,14 @@ def _find_radii(self, num_ens, cube_lead_times=None):
                 Required neighbourhood sizes.
         """
         if cube_lead_times is None:
-            radii = self.adjust_nsize_for_ens(num_ens, self.radii)
+            radii = self.radii
```

Comment:
This could be:
```
if cube_lead_times is None:
    return self.radii
radii = (np.interp.....
....
return radii
```

The else is then implicit and the later code only reached if the if statement is not true. I don't really mind either way, but Ben F tends to prefer this style.

Reply:
Done.